client/src/interfaces/eventData.tsx

```diff
@@ -0,0 +1,10 @@
+export default interface eventData {
+  name: string;
+  description: string;
+  img: string;
+  status: 'in-progress' | 'closed' | 'upcoming';
+  startTime: string;
+  profileImage: string;
+  username: string;
+// eslint-disable-next-line semi, no-extra-semi
+};
```

Comment:
it's great to use interfaces
but i think we should follow a convention when we name it

i suggest to name the interfaces with this convention
`IEvent` instead of `eventData`

Note: `I` in `IEvent` stands for `interface`